src/core/server/core_usage_stats/types.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * @internal
+ *
+ * CoreUsageStats are collected over time while Kibana is running. This is related to CoreUsageData, which is a superset of this that also
+ * includes point-in-time configuration information.
+ * */
+export interface CoreUsageStats {
+  apiCalls?: {
+    savedObjectsImport?: {
+      total: number;
+      kibanaRequest: {
+        yes: number;
+        no: number;
+      };
```

Comment:
Each of these usage stats objects now contain a `kibanaRequest` field, which we use to determine whether this request was made from the Kibana client (first-party) or not (third-party).

Reply:
We may want to be able to count how many imports are Kibana requests. We also may want to count how many imports are _not_ Kibana requests. We could get rid of the separate yes/no fields, but then any aggregations can get complex.

Yes total is a bit redundant as you could derive that from yes+no, but again the focus is on making aggregations and analyses simpler. Since this is not time-series data, we can be a bit liberal with our field usage IMO =)